app/controllers/api/v1/code_schools_controller.rb

```diff
@@ -16,7 +16,7 @@ def create
       end
 
       def show
-        school = CodeSchool.find_by(id: params[:id])
```

Comment:
Actually, you were good to go with the `find_by` method.  Reason being, if `find` does not find the id in question, it blows up and throws and error.

However, if `find_by` does not find what you're looking for, it returns `nil`.